content/topics/reopening-pull-request/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+_db_id: 
+content_type: topic
+ready: true
+title: Reopening a pull request
+---
+
+## Handling requested changes on Pull Requests
```

Comment:
just check for consistency in capitalisation of "pull request" sometimes you have it capitalised and sometimes not